build/charts/theia/provisioning/datasources/migrators/000005_0-5-0.up.sql

```diff
@@ -0,0 +1,13 @@
+--Add aggregated TAD columns
```

Comment:
We may also need to change the filename `000005_0-5-0.up.sql` in the future when this PR merged. Just mentioning it for now, you can do it later before merging it.